libs/README.md

```diff
@@ -0,0 +1,153 @@
+ - `[-bin]` is added for assets that contain executable binaries. For example, for `libjpeg` those will be `cjpeg`, `djpeg`, `jpegtran`, etc. Naturally, those binaries are built as Wasm modules.
+ - `LIBRARY_VERSION` is the concrete version of that library.
+ - `WASI_SDK_VERSION` is the version of `wasi-sdk` used to build the asset.
+
+A library asset will contain the header(s), static archive(s) and pkg-config file(s) for this library. The headers are in `include`, while the rest goes in `lib/wasm32-wasi`. For example for `libjpeg` we get:
+
+```shell-session
+$$ tar -ztvf libjpeg-2.1.5.1-wasi-sdk-19.0.tar.gz
+
+drwxr-xr-x root/root         0 2023-03-09 09:52 include/
+-rw-r--r-- root/root     15864 2023-03-09 09:52 include/jerror.h
+-rw-r--r-- root/root      1091 2023-03-09 09:52 include/jconfig.h
+-rw-r--r-- root/root     50281 2023-03-09 09:52 include/jpeglib.h
+-rw-r--r-- root/root     14192 2023-03-09 09:52 include/jmorecfg.h
+drwxr-xr-x root/root         0 2023-03-09 09:52 lib/
+drwxr-xr-x root/root         0 2023-03-09 09:52 lib/wasm32-wasi/
+drwxr-xr-x root/root         0 2023-03-09 09:52 lib/wasm32-wasi/pkgconfig/
+-rw-r--r-- root/root       246 2023-03-09 09:52 lib/wasm32-wasi/pkgconfig/libjpeg.pc
+-rw-r--r-- root/root    470134 2023-03-09 09:52 lib/wasm32-wasi/libjpeg.a
+```
+
+If there is also a `-bin` asset it will usually have just a `bin` folder with the binaries. For example, for `libjpeg-bin` we get:
+
+```shell-session
+$$ tar -ztvf libjpeg-bin-2.1.5.1-wasi-sdk-19.0.tar.gz
+
+drwxr-xr-x root/root         0 2023-03-09 09:52 bin/
+-rwxr-xr-x root/root     41328 2023-03-09 09:52 bin/rdjpgcom
+-rwxr-xr-x root/root    390627 2023-03-09 09:52 bin/jpegtran
+-rwxr-xr-x root/root     40453 2023-03-09 09:52 bin/wrjpgcom
+-rwxr-xr-x root/root    213619 2023-03-09 09:52 bin/djpeg
+-rwxr-xr-x root/root    360896 2023-03-09 09:52 bin/cjpeg
+```
+
+# How to use
+
+To see how to use the released libraries with an end-to-end demo, take a look at the [libs/examples](./examples/) folder.
+
+In short, you have two basic options:
+
+1. Download and extract inside a `${WASI_SDK_PATH}/share/wasi-sysroot` folder. In this way you shouldn't need to setup include or link paths. This is most useful when you use a download a fresh wasi-sdk setup as part of your build process, but might get tricky if you overwrite those in your local development wasi-sdk environment.
+
+2. Download and extract the files in some folder (e.g. `${SOME_TEMP_BUILD_FOLDER}/dependencies`) then add respectively the `include` and `lib/wasm32-wasi` folders to include dirs and library dirs. (e.g. `export CFLAGS="${CFLAGS} -I${SOME_TEMP_BUILD_FOLDER}/dependencies/include"` and `export CFLAGS="${CFLAGS} -L${SOME_TEMP_BUILD_FOLDER}/dependencies/lib/wasm32-wasi"`)
+
+ If your build relies on `pkg-config`, to avoid it picking up headers and versions from your build machine, you could set it up for cross-compilation like this:
```

Comment:
```suggestion
 If your build relies on `pkg-config`, to avoid picking up headers and versions from your build machine, you could set it up for cross-compilation like this:
```